appengine-java8/tasks/snippets/src/main/java/com/example/task/Snippets.java

```diff
@@ -0,0 +1,314 @@
+/*
+ * Copyright 2018 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.appengine;
+
+import com.google.cloud.tasks.v2.AppEngineHttpRequest;
+import com.google.cloud.tasks.v2.AppEngineRouting;
+import com.google.cloud.tasks.v2.CloudTasksClient;
+import com.google.cloud.tasks.v2.HttpMethod;
+import com.google.cloud.tasks.v2.LocationName;
+import com.google.cloud.tasks.v2.Queue;
+import com.google.cloud.tasks.v2.QueueName;
+import com.google.cloud.tasks.v2.RateLimits;
+import com.google.cloud.tasks.v2.RetryConfig;
+import com.google.cloud.tasks.v2.Task;
+import com.google.cloud.tasks.v2.TaskName;
+import com.google.cloud.tasks.v2.UpdateQueueRequest;
+import com.google.protobuf.ByteString;
+import com.google.protobuf.Duration;
+import java.nio.charset.Charset;
+
+public class Snippets {
+  //  [START taskqueues_using_yaml]
```

Comment:
Should these follow the new format and be in different files?

Reply:
done.